src/data/event-qr-code-guide.json

```diff
@@ -11,12 +11,12 @@
                 "url": "/assets/img/event-qr-code-guide/nodejs-website.png",
                 "alt": "node.js installation"
             },
-            "text2": "<hr/> We explicitly point out that you should <b>not</b> set the checkmark at the following pop-up window. The installation can also be done without this additional option. <br/> <br/> <center> <b>Window text : </b> <br/> <em>Some npm modules need to be compiled from C/C++ when installing. If you want to be able to install such modules, some tools (Python and Visual Studio Build Tools) need to be installed. <br> <br> [ ] Automatically install the necessary tools. Note that this will also install Chocolatey. The script will pop-up in a new window after the installation completes. <br> <br> Alternatively, follow the instructions at https://github.com/nodejs/node-gyp#on-windows to install the dependencies yourself. </em> </center>  ",
+            "text2": "<hr/> We explicitly point out that you should <b>not</b> set the checkmark at the following pop-up window. The installation can also be done without this additional option. <br/> <br/> <div> <b>Window text : </b> <br/> <em>Some npm modules need to be compiled from C/C++ when installing. If you want to be able to install such modules, some tools (Python and Visual Studio Build Tools) need to be installed. <br> <br> [ ] Automatically install the necessary tools. Note that this will also install Chocolatey. The script will pop-up in a new window after the installation completes. <br> <br> Alternatively, follow the instructions at https://github.com/nodejs/node-gyp#on-windows to install the dependencies yourself. </em> </div>  ",
```

Comment:
Center has been swapped with <div> but without styles, this will not be centered

Reply:
@brianebeling 

Moving some of the text to left-alignment in this case is an improvement. The center formatting was quite strange previously.